pyproject.toml

```diff
@@ -27,10 +46,43 @@ pytest-watch = "^4.2.0"
 django-filter = "^2.4.0"
 pytest-mock = "^3.5.1"
 
+
+[tool.poetry.extras]
+docs = [
+    "mkdocs",
+    "mkdocs-markdownextradata-plugin",
+    "mkdocs-material",
+    "mkdocs-minify-plugin",
+    "pymdown-extensions",
+    "markdown",
+]
+
+
 [build-system]
 requires = ["poetry-core>=1.0.0", "setuptools"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.black]
+line-length = 100
```

Comment:
Although I personally use a line length of 100, strawberry's defaults is to use 80. Probably we shouldn't change this